model/src/main/scala/uk/gov/ons/addressIndex/model/db/index/NationalAddressGazetteer.scala

```diff
@@ -0,0 +1,100 @@
+package uk.gov.ons.addressIndex.model.db.index
+
+import uk.gov.ons.addressIndex.model.server.response._
+
+case class NationalAddressGazetteer(
```

Comment:
Maybe we should move these into 
`uk.gov.addressIndex.model.db.index.format` to provide context?